uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/repositories/ChangeRequestServiceImpl.java

```diff
@@ -1333,4 +1334,61 @@ JGitPathImpl createJGitPathImpl(final JGitFileSystem fs,
                                    objectId,
                                    isRealPath);
     }
-}
\ No newline at end of file
+
+    @Override
+    public List<ChangeRequestCommit> getCommits(final String spaceName,
```

Comment:
Consider creating at least one unit test for this public method.

Reply:
Done